### ISSUE TITLE
GCC compiler needs update

### ISSUE THREAD

**Issue by PeyTy** [GreenteaOS/greenteasdk#2]
New version of GCC+mingw is out?

**Comment by xdev31** [comment]
Yep, ReactOS uses https://svn.reactos.org/RosBE-Sources/rosbe_2.1/ those files <<

This is repo for GreenteaSDK and I think it shouldn't contain any gcc

**Comment by PeyTy** [comment]
Isn't this SDK based on RosBE? We need just RosBE-alike toolkit for Greentea with built-in gcc and some basic tools (rc compiler, ninja etc).

**Comment by xdev31** [comment]
Yep, this is the repo for rosbe scripts in windows

**Comment by xdev31** [comment]
Question: Why don't you upgrade mingw and others?
Answer: Because we would keep compatibility under ReactOS and Windows 2003. 

If I will upgrade tools, I need to rewrite every scripts

**Comment by PeyTy** [comment]
So basically newest mingw is incompatible with ROS/2003?

**Comment by xdev31** [comment]
@PeyTy  I never try to run them in 2003

**Comment by PeyTy** [comment]
Maybe mingw generates incompatible executables. This may actually happen.
Don't know why scripts must be changed...
Okay if it is a big troube then let freeze this as a long term goal.

**Comment by xdev31** [comment]
For scripts, we will need to test them again and after that I will edit if there will be any errors...

I don't know how I can keep update with mingw
